server/controllers/auth.controller.js

```diff
@@ -18,5 +21,43 @@ module.exports = {
       }
     })
     .catch(error => res.status(401).send(error));
+  },
+  activation(req, res) {
+    let token = req.params.token;
+    let decoder;
+    try {
+      decoder = jwt.verify(token, secret.key);
+    } catch (err) {
+      res.status(498).send(message.linkNotValid);
+    }
+    User.findOne({
+      where: {
+        id: decoder.id
+      }
+    })
+    .then(user => {
+      if (!user) {
+        res.status(404).send(message.userNotValid);
+      } else {
+        if (user.is_activate) {
+          res.status(200).send(message.linkAlreadyActivated);
```

Comment:
this message don't use to success activation, it used to link which was double clicked by user in email, should i still change it?